pkg/apis/core/v1beta1/kubefedconfig_types.go

```diff
@@ -36,6 +36,8 @@ type KubeFedConfigSpec struct {
 	// +optional
 	ClusterHealthCheck *ClusterHealthCheckConfig `json:"clusterHealthCheck,omitempty"`
 	// +optional
+	Concurrency *ConcurrencyConfig `json:"concurrency,omitempty"`
```

Comment:
How about we move the concurrency settings under the respective controller config, e.g. add sync controller concurrency config in `SyncControllerConfig`?

Reply:
Thanks for the advice, it seems to be more clear indeed. I'll make this change.

Reply:
Thanks @zqzten!